test/features/my_academics.feature

```diff
@@ -0,0 +1,7 @@
+Feature: My Academics Enrollment Page
+  User should be able to click on schedule of classes,add,drop,swap,edit classes
```

Comment:
@mDibyo I'm not sure if these initial test cases are intended to be used as examples - but if so, it might be useful to clarify that some of these features are only available based on properties of the user.  For example, a user might not be able to fully utilize all the features of the enrollment card if they have a hold on their account.